lib/index.js

```diff
@@ -45,18 +45,19 @@
  * The main interface is the {@link analyze} function.
  * @module escope
  */
-"use strict";
-
 /* eslint no-underscore-dangle: ["error", { "allow": ["__currentScope"] }] */
 
-const assert = require("assert");
+import { readFileSync } from "fs";
```

Comment:
This will crash our [eslint demo](https://eslint.org/demo) build.

I tried to copy this eslint-scope version over the one in `node-modules` under [src/js/demo](https://github.com/eslint/website/tree/master/src/js/demo), and then `npm run build:webpack` failed with:

```
ERROR in ./src/js/demo/node_modules/eslint-scope/dist/eslint-scope.cjs 5:9-22
Module not found: Error: Can't resolve 'fs' in '...\src\js\demo\node_modules\eslint-scope\dist'
```

Reply:
I have removed the `fs` dependency, but currently the Rollup config I added is treating the following as external and is not bundling them: `assert`, `estraverse`, and `esrecurse`.

Since you use `external` in [espree](https://github.com/eslint/espree/blob/main/rollup.config.js), I figured you were leaving it to consumers to bundle, but if the demo is still not working after dropping `fs` and you want, I could add `@rollup/plugin-commonjs`, etc. to have the latter two bundled (with the disadvantage that they will be baked-in versions) and add something like [`rollup-plugin-unassert`](https://github.com/unassert-js/rollup-plugin-unassert) or [`rollup-plugin-node-builtins`](https://github.com/calvinmetcalf/rollup-plugin-node-builtins) to handle `assert`. Otherwise, I'm not sure how to proceed.

Reply:
> I have removed the `fs` dependency, but currently the Rollup config I added is treating the following as external and is not bundling them: `assert`, `estraverse`, and `esrecurse`.

That should be fine, we don't intend to bundle any external modules.

The currently published eslint-scope is a set of cjs modules from the `lib/` directory, so consumers already handle `require("assert")`, `require("estraverse")`, and `require("estraverse")` while bundling (e.g., eslint demo uses `node-polyfill-webpack-plugin` to handle `assert`). I think the new elint-scope will be basically the same thing in that regard, just packed in one module `dist/eslint-scope.cjs`.